lib/content-services/src/lib/search/components/search-date-range/search-date-range.component.html

```diff
@@ -1,49 +0,0 @@
-<form [formGroup]="form" novalidate (ngSubmit)="apply(form.value, form.valid)">
-    <mat-form-field>
-        <mat-label>{{ 'SEARCH.FILTER.RANGE.SELECT-FROM-DATE' | translate }}</mat-label>
-        <input matInput
-                (input)="forcePlaceholder($event)"
-                [formControl]="from"
-                [errorStateMatcher]="matcher"
-                placeholder="{{ 'SEARCH.FILTER.RANGE.FROM-DATE' | translate }}"
-                [attr.aria-label]="'SEARCH.FILTER.RANGE.FROM-DATE' | translate"
-                [matDatepicker]="fromDatepicker"
-                [max]="fromMaxDate"
-                (dateChange)="onChangedHandler($event, from)"
-                data-automation-id="date-range-from-input">
-        <mat-datepicker-toggle matSuffix [for]="fromDatepicker" data-automation-id="date-range-from-date-toggle"></mat-datepicker-toggle>
-        <mat-datepicker #fromDatepicker color="accent"></mat-datepicker>
-        <mat-error *ngIf="from.invalid" data-automation-id="date-range-from-error">
-            {{ getFromValidationMessage() | translate: { requiredFormat: datePickerFormat } }}
-        </mat-error>
-    </mat-form-field>
-
-    <mat-form-field>
-        <mat-label>{{ 'SEARCH.FILTER.RANGE.SELECT-TO-DATE' | translate }}</mat-label>
-        <input matInput
-                (input)="forcePlaceholder($event)"
-                [formControl]="to"
-                [errorStateMatcher]="matcher"
-                placeholder="{{ 'SEARCH.FILTER.RANGE.TO-DATE' | translate }}"
-                [attr.aria-label]="'SEARCH.FILTER.RANGE.TO-DATE' | translate"
-                [matDatepicker]="toDatepicker"
-                [min]="from.value"
-                [max]="maxDate"
-                (dateChange)="onChangedHandler($event, to)"
-                data-automation-id="date-range-to-input">
-        <mat-datepicker-toggle matSuffix [for]="toDatepicker" data-automation-id="date-range-to-date-toggle"></mat-datepicker-toggle>
```

Comment:
that variable seems still referencing to that removed element, please remove that variable as this element does not exist anymore: 
<img width="1112" alt="image" src="https://github.com/Alfresco/alfresco-ng2-components/assets/115619721/d526b302-f1c1-4cfc-86a5-127c54b910e2">

Reply:
Removed date-range-filter.page.ts